fims-api/swagger.yaml

```diff
@@ -0,0 +1,176 @@
+swagger: "2.0"
+info:
```

Comment:
fims-api  is archaic and not part of UAM.  Do not add

Reply:
Shall remove uam-apis/fims-api